### PR TITLE
First home layouts

### PR DESCRIPTION
**Pull request by jdesiloniz**
This PR includes changes in the current Jekyll structure:

- Header and footer partials, already handling real data.
- Using filenames from data collections as unique ids for team members.

Could you please review, @jorgegalindocruces? Thanks!

### CODE REVIEW

File: _content/_data/footer-nav.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- categoryName: General
+  links:
+   - title: Services
```
Comment (jorgegalindocruces) [inline _content/_data/footer-nav.yml]: Here we have to add Open Source that it will be an external website

File: _content/_data/partners.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+- url: http://www.lightbend.com
+  image: /assets/img/partners/lightbend.svg
+
+- url: https://databricks.com
+  image: /assets/img/partners/databrick.svg
+
+- url: https://www.confluent.io
```
Comment (jorgegalindocruces) [inline _content/_data/partners.yml]: These links are internals links but dont worry about that for the moment

File: _content/_data/theme.yml [hunk 3]
```diff
@@ -59,14 +59,14 @@ recent_project_es: Proyectos reciente
 download_app_title: Download Today
```
Comment (jorgegalindocruces) [inline _content/_data/theme.yml]: WTF?

File: _content/_posts/2016-10-27-practical-scalacheck.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "Tips and Techniques for using ScalaCheck"
 meta_desc: "Noel Markham introduces techniques for making the most out of using ScalaCheck, when to use Gen and Arbitrary classes, and a great way to cut down your boilerplate."
 featured_image: /assets/img/company/engineering-asset@2x.jpg
 masthead_image: /assets/img/company/engineering-asset@2x.jpg
-author: noel
+author: 16-noel
```
Comment (jorgegalindocruces) [inline _content/_posts/2016-10-27-practical-scalacheck.md]: This is mandatory?

File: _themes/fortysevendeg/_includes/_doc_head.html [hunk 5]
```diff
@@ -3,54 +3,8 @@
...
-            Typekit.load();
-        } catch (e) {}
-    </script>
-
-    <!-- AddThisEvent -->
-    <script type="text/javascript" src="https://addthisevent.com/libs/1.6.0/ate.min.js"></script>
-    <link rel="alternate" type="application/rss+xml" href="{{ _site_url }}/feed" title="{{ _site_name }} RSS Feed">
-    {{ theme:partial src="_analytics" }}
-    {{ highlight:head }}
+    <meta charset="UTF-8">
```
Comment (jorgegalindocruces) [inline _themes/fortysevendeg/_includes/_doc_head.html]: Be careful with the doc_head, is now very good done to be SEO friendly and meta tags like twitter, facebook and everything of that, has to be here.

File: _content/_posts/2016-10-27-practical-scalacheck.md (hunk 4 above)
Comment (jdesiloniz) [inline _content/_posts/2016-10-27-practical-scalacheck.md]: Right on this. Switched to using a "member" variable in each of the team members.

File: _themes/fortysevendeg/_includes/_doc_head.html (hunk 5 above)
Comment (jdesiloniz) [inline _themes/fortysevendeg/_includes/_doc_head.html]: Brought back the meta tags from the previous version to the new header.

**Comment by jorgegalindocruces** [comment]
LGTM!

**Pull request merged**